src/clue/adafruit_clue.py

```diff
@@ -199,15 +199,65 @@ class Clue:  # pylint: disable=too-many-instance-attributes, too-many-public-met
     RAINBOW = (RED, ORANGE, YELLOW, GREEN, BLUE, PURPLE)
 
     def __init__(self):
+        self._a = False
+        self._b = False
+        self.__pressed_buttons = set()
         self._pixel = neopixel.NeoPixel(
             pin=CONSTANTS.CLUE_PIN, n=1, pixel_order=neopixel.RGB
         )
 
+    @property
+    def button_a(self):
+        """``True`` when Button A is pressed. ``False`` if not.
+        This example prints when button A is pressed.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              if clue.button_a:
+                  print("Button A pressed")
+        """
+        return self._a
+
+    @property
+    def button_b(self):
+        """``True`` when Button B is pressed. ``False`` if not.
+        This example prints when button B is pressed.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              if clue.button_b:
+                  print("Button B pressed")
+        """
+        return self._b
+
+    def __update_button(self, button, value):
+        if button == "button_a":
+            if value:
+                self.__pressed_buttons.add("A")
+            self._a = value
+        elif button == "button_b":
+            if value:
+                self.__pressed_buttons.add("B")
+            self._b = value
+
+    @property
+    def were_pressed(self):
+        """Returns a set of the buttons that have been pressed.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              print(clue.were_pressed)
+        """
+        ret = self.__pressed_buttons.copy()

```

Comment:
I see here that this is probably why you opted for a set. But, it might be more worth it to use a dict for clarity (?)

Reply:
If we use dictionary, it makes it seem that we are mapping strings to other objects, but in this case (if I understand correctly) we only map to boolean (true/false).
What do you think?

Reply:
I used a set because that is how it is returned in the actual CLUE